static_src/components/user_list.jsx

```diff
@@ -126,47 +126,49 @@ export default class UserList extends React.Component {
         { this.documentation }
         <ComplexList>
           { this.props.users.map((user) => {
-            let actions;
-            if (this.props.onRemove) {
-              let button = <span></span>;
-              if (this.props.currentUserAccess) {
-                if (this.props.userType === 'org_users') {
-                  buttonText = 'Remove User From Org';
-                } else if (this.props.userType === 'space_users') {
-                  buttonText = 'Remove All Space Roles';
+            if (!!user.username) {
+              let actions;
+              if (this.props.onRemove) {
+                let button = <span></span>;
+                if (this.props.currentUserAccess) {
+                  if (this.props.userType === 'org_users') {
+                    buttonText = 'Remove User From Org';
+                  } else if (this.props.userType === 'space_users') {
+                    buttonText = 'Remove All Space Roles';
+                  }
+                  button = (
+                    <Action
+                      style="base"
+                      clickHandler={ this._handleDelete.bind(this, user.guid) }
```

Comment:
Calling a `bind` directly inside the render function has performance implications, as the function will be rebound (creating a memory allocation + running the garbage collector) every time the render function runs. Additionally, since we are always assigning a new value to the `clickHandler` property, the component will always be re-rendered.

We should probably refactor this. I think simplest way would be to break the logic contained within the `map`'s callback out into a component.